app.py

```diff
@@ -12,11 +13,11 @@ def main():
 
     for arg in args:
         if '--server' in arg:
-            Server().start(9999, '0.0.0.0')
+            Server().start(9999, 'localhost')
```

Comment:
This is a target, so replace '0.0.0.0' with '127.0.0.1' means that this server will just accept connections from localhost, what to me make sense keep your change right now :+1: